cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/TransactionManagerDebuggerMain.java

```diff
@@ -229,9 +230,11 @@ private void printUsage(boolean error) {
     } else {
       pw = new PrintWriter(System.out);
     }
+    
+    private static final String toolName = "tx-debugger" + OSDetector.isWindows() ? ".bat" : ".sh";
```

Comment:
sorry, don't need to use `private static final` prefix, when its used from a method.
Just if it was declared as a field of the class.